main.go

```diff
@@ -52,6 +55,20 @@ func getConfig() (*config, error) {
 	gitlabToken := os.Getenv("GITLAB_TOKEN")
 	gitlabBaseURL := os.Getenv("GITLAB_BASE_URL")
 	integrationDirectory := os.Getenv("INTEGRATION_DIRECTORY")
+	osToEntGithubToken  := os.Getenv("OS_SYNC_GITHUB_TOKEN")
+	logLevel, found := os.LookupEnv("INTEGRATION_TEST_RUNNER_LOG_LEVEL")
+
+	if found {
+		lvl, err := log.ParseLevel(logLevel)
+		if err != nil {
+			log.Infof("Failed to parse the 'INTEGRATION_TEST_RUNNER_LOG_LEVEL' variable, defaulting to 'InfoLevel'")
```

Comment:
Defaulting to info level... but where are you setting it? ;)

Reply:
Yeh, Info is the default, however, explicit is always better imo. Fixing.

Reply:
It's that, or removing the explicit one below.

Reply:
I added the explicit case for both.

Reply:
Exactly. I know implicit is logging, but as you are doing it below for the else case... I would also vote for explicit (consistency, you know).

If any, you could set it unconditionally to info at the start, and then only change it in case the user used the variable and it is a valid level.

Reply:
Good idea.